examples/fastai_simple.py

```diff
@@ -0,0 +1,78 @@
+"""
+Optuna example that demonstrates a pruner for fastai.
+
+In this example, we optimize the hyperparameters of a convolutional neural network for hand-written
+digit recognition in terms of validation accuracy. The network is implemented by fastai and
+evaluated on MNIST dataset. Throughout the training of neural networks, a pruner observes
+intermediate results and stops unpromising trials.
+Note that this example will take longer than the other examples
+as this uses the entire MNIST dataset.
+
+You can run this example as follows, pruning can be turned on and off with the `--pruning`
+argument.
+    $ python fastai_integration.py [--pruning]
+"""
+
+import argparse
+from functools import partial
+
+from fastai.vision import ImageDataBunch
+from fastai.vision import Learner
+from fastai.vision import URLs
+from fastai.vision import accuracy
+from fastai.vision import simple_cnn
+from fastai.vision import untar_data
+
+import optuna
+from optuna.integration import FastAIPruningCallback
+
+
+BATCHSIZE = 128
+EPOCHS = 10
+
+
+path = untar_data(URLs.MNIST_SAMPLE)
+data = ImageDataBunch.from_folder(path, bs=BATCHSIZE)
+
+
+def objective(trial: optuna.trial.Trial) -> float:
```

Comment:
```suggestion
def objective(trial):
```
Please remove the type hint for simplicity (c.f.,  https://github.com/optuna/optuna/wiki/Coding-Style-Conventions#dont-use-type-hint-in-examples).

Reply:
Ah, I forgot to remove this after the development. Thank you.